docs/proposals/multiple-schedulers.md

```diff
@@ -152,6 +152,11 @@ which the client has set a scheduler annotation that does not correspond to a ru
 2. Dynamic launching scheduler(s) and registering to admission controller (as an external call). This also
 requires some work on authorization and authentication to control what schedulers can write the /binding
 subresource of which pods.
+3. Optimize the behaviors of priority functions in multi-scheduler scenario. In the case where multiple schedulers have
+the same predicate and priority functions (for example, when using multiple schedulers for parallelism rather than to
+customize the scheduling policies), all schedulers would tend to pick the same node as "best". To solve this problem,
```

Comment:
Sorry, I just realized the statement "all schedulers would tend to pick the same node as "best"" needs a little more explanation.

Maybe say something like `all schedulers would tend to pick the same node as "best" when scheduling identical pods and therefore would be likely to conflict in the optimistic concurrency control.`

(You're right that some priority functions will return the same value for all pods, like LeastRequestedPriority. But some priority functions depend on properties of the pod, like the ones in selector_spreading.go. So it is better to be clear and say "when scheduling identical pods.") This is still a realistic scenario because you could imagine a large ReplicationController, which by definition generates identical pods, where half of the pods go to one scheduler and half go to the other scheduler.